simulation_lab/SecurityHeaderAdder.py

```diff
@@ -0,0 +1,49 @@
+def SecurityHeaderAdder(get_response):
```

Comment:
Maybe add more documentation to this file explaining how it works, what it's implications are, and where we can learn more about content-security-policy (if you have a good link)